### ISSUE TITLE
Build error with 0.9.1.0 on Debian Wheezy

### ISSUE THREAD

**Issue by jonkri** [lpsmith/postgresql-libpq#31]
Hi!

Trying to build postgresql-libpq-0.9.1.0 (in a context which built the previous version successfully), I'm getting an error like this:

    Configuring postgresql-libpq-0.9.1.0...
    Building postgresql-libpq-0.9.1.0...
    Preprocessing library postgresql-libpq-0.9.1.0...
    LibPQ.hsc:215:28: fatal error: noticehandlers.h: No such file or directory
    compilation terminated.
    compiling dist/build/Database/PostgreSQL/LibPQ_hsc_make.c failed (exit code 1)
    command was: /usr/bin/gcc -c dist/build/Database/PostgreSQL/LibPQ_hsc_make.c -o dist/build/Database/PostgreSQL/LibPQ_hsc_make.o -fno-stack-protector -D__GLASGOW_HASKELL__=708 -Dlinux_BUILD_OS=1 -Dx86_64_BUILD_ARCH=1 -Dlinux_HOST_OS=1 -Dx86_64_HOST_ARCH=1 -I/usr/include/postgresql -Icbits -I/opt/ghc/7.8.3/lib/ghc-7.8.3/bytestring-0.10.4.0/include -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/base-4.7.0.1/include -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/integer-gmp-0.5.1.0/include -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/include -Idist/build/autogen -include dist/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/include/

**Comment by dzhus** [comment]
I'm seeing the same error on our build server starting today:

```
Failed to install postgresql-libpq-0.9.1.0
Build log ( /home/carma/carma/srv/.cabal-sandbox/logs/postgresql-libpq-0.9.1.0.log ):
[1 of 1] Compiling Main             ( /tmp/postgresql-libpq-0.9.1.0-21968/postgresql-libpq-0.9.1.0/dist/dist-sandbox-819d318a/setup/setup.hs, /tmp/postgresql-libpq-0.9.1.0-21968/postgresql-libpq-0.9.1.0/dist/dist-sandbox-819d318a/setup/Main.o )
Linking /tmp/postgresql-libpq-0.9.1.0-21968/postgresql-libpq-0.9.1.0/dist/dist-sandbox-819d318a/setup/setup ...
Configuring postgresql-libpq-0.9.1.0...
Building postgresql-libpq-0.9.1.0...
Preprocessing library postgresql-libpq-0.9.1.0...
LibPQ.hsc:215:28: fatal error: noticehandlers.h: No such file or directory
compilation terminated.
compiling dist/dist-sandbox-819d318a/build/Database/PostgreSQL/LibPQ_hsc_make.c failed (exit code 1)
command was: /usr/bin/gcc -c dist/dist-sandbox-819d318a/build/Database/PostgreSQL/LibPQ_hsc_make.c -o dist/dist-sandbox-819d318a/build/Database/PostgreSQL/LibPQ_hsc_make.o -fno-stack-protector -D__GLASGOW_HASKELL__=708 -Dlinux_BUILD_OS=1 -Dx86_64_BUILD_ARCH=1 -Dlinux_HOST_OS=1 -Dx86_64_HOST_ARCH=1 -I/usr/include/postgresql -Icbits -I/opt/ghc/7.8.3/lib/ghc-7.8.3/bytestring-0.10.4.0/include -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-sandbox-819d318a/build/autogen/cabal_macros.h -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-sandbox-819d318a/build/autogen/cabal_macros.h -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-sandbox-819d318a/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/base-4.7.0.1/include -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-sandbox-819d318a/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/integer-gmp-0.5.1.0/include -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-s
 andbox-819d318a/build/autogen/cabal_macros.h -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-sandbox-819d318a/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/include -Idist/dist-sandbox-819d318a/build/autogen -include dist/dist-sandbox-819d318a/build/autogen/cabal_macros.h -I/opt/ghc/7.8.3/lib/ghc-7.8.3/include/
```

**Comment by lpsmith** [comment]
Yeah,  my bad.   I just merged #32 and pushed version v0.9.1.1 to hackage.

**Comment by jonkri** [comment]
Thanks! I can confirm that it builds successfully in my environment now.
